pkg/resources/grant_ownership_acceptance_test.go

```diff
@@ -884,6 +884,7 @@ resource "snowflake_grant_account_role" "test" {
 }
 `, accountRoleName, databaseName, userName)
 
+	// TODO: build these configs from builders
```

Comment:
Do we have a ticket for this?

Reply:
Yes, missed this one (but it was listed both in the readme and the issue with improvements. Added the number.